addons/mail/models/mail_scheduled_message.py

```diff
@@ -150,6 +152,29 @@ def write(self, vals):
             self.env.ref('mail.ir_cron_post_scheduled_message')._trigger(fields.Datetime.to_datetime(new_scheduled_date))
         return res
 
+    # ------------------------------------------------------
+    # Compute Methods
+    # ------------------------------------------------------
+
+    @api.depends('model', 'partner_ids', 'res_id')
+    def _compute_notified_bcc(self):
+        """ When being in monorecord comment mode, compute 'bcc' which are
+        followers that are going to be 'silently' notified by the message. """
+        for composer in self:
+            record = self.env[composer.model].browse(composer.res_id)
+            params = json.loads(composer.notification_parameters or '{}')
+            recipients_data = self.env['mail.followers']._get_recipient_data(
```

Comment:
Meh, one complex query / record to display ... really don't like that. It is ok if it stay as a one-shot field on a specific view, just be sure it is never displayed "in batch".

Reply:
I tried adding a batch behavior to this compute but I am not a big fan of what I did. The notified_bcc should only be displayed in the case of the `mail.scheduled.message` form view, and don't really see it being added to any list view. But I would be glad to discuss this one if you want.